bridge/bridge/api/static/openapi/openapi.yaml

```
@@ -0,0 +1,919 @@
+        Address used for filtering wrap or unwrap requests and errors.
+        The exact format of addresses depends on the configured network (e.g. Ethereum, Symbol, NEM).
+        Clients should validate addresses according to the blockchain in use.
+      schema:
+        type: string
+        example: 0x12ab34cd56ef7890ab12cd34ef567890ab12cd34
+
+    TransactionHash:
+      name: transaction_hash
+      in: path
+      required: true
+      description: Transaction hash used to filter wrap or unwrap requests and errors.
+      schema:
+        type: string
+        pattern: '^[0-9A-Fa-f]{64}$'
+        example: 3f5b8f9a2bb2f6a9c3d2b4f0a1e2c3d4e5f6a7b8c9d0e1f2a3b4c5d6e7f8a90
+
+    Offset:
+      name: offset
+      in: query
+      required: false
+      description: Number of items to skip for pagination.
+      schema:
+        type: integer
+        minimum: 0
+        default: 0
+
+    Limit:
+      name: limit
+      in: query
+      required: false
+      description: Maximum number of items to return.
+      schema:
+        type: integer
+        minimum: 0
+        default: 25
+
+    Sort:
+      name: sort
+      in: query
+      required: false
+      description: |
+        Sort order for results by request transaction height. Values: `1` (ascending) or `0` (descending).
+      schema:
+        type: integer
+        enum:
+          - 1
+          - 0
+        default: 1
+
+    PayoutStatus:
+      name: payout_status
+      in: query
+      required: false
+      description: Filter by payout processing status (0 – unprocessed, 1 - sent, 2 - completed, 3 - failed).
+      schema:
+        type: integer
+        enum:
+          - 0
+          - 1
+          - 2
+          - 3
+
+  schemas:
+    BridgeConfiguration:
+      type: object
+      description: Contains operational bridge configuration.
+      required:
+        - nativeNetwork
+        - wrappedNetwork
+        - enabled
+      properties:
+        nativeNetwork:
+          $ref: '#/components/schemas/NetworkConfig'
+        wrappedNetwork:
+          $ref: '#/components/schemas/NetworkConfig'
+        enabled:
+          type: boolean
+
+    NetworkConfig:
+      type: object
+      description: Network configuration details for either the native or wrapped chain.
+      required:
+        - blockchain
+        - network
+        - bridgeAddress
+        - tokenId
+        - defaultNodeUrl
+        - explorerUrl
+      properties:
+        blockchain:
+          description: Blockchain type on which this network operates.
+          type: string
+          enum:
+            - nem
+            - symbol
+            - ethereum
+          example: ethereum
+        network:
+          description: Network environment (testnet or mainnet).
+          type: string
+          enum:
+            - testnet
+            - mainnet
+          example: testnet
+        bridgeAddress:
+          description: Address monitored for incoming wrap or unwrap requests.
```

```suggestion
          description: Address monitored for incoming wrap (when on the native network) or unwrap (when on the wrapped network) requests.
```

Otherwise it looks like the same address can take care of both wrap and unwrap requests.

---

Changed.